cli/index.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+const { Command } = require('commander');
+const program = new Command();
+const { execSync, spawn } = require('child_process');
+const net = require('net');
+const fs = require('fs');
+
+function run(command) {
+    try {
+        console.log(`> ${command}`)
+        return execSync(command, { stdio: 'pipe' }).toString();
+    } catch (error) {
+        console.error(error.message);
+        process.exit(1);
+    }
+}
+program
+    .name('nest')
+    .description('Yet another tilde caddy manager')
```

Comment:
```suggestion
    .description('A command-line interface for Nest tools and services')
```
nest cli description (something like this, doesn't have to be exactly)